docs/dev/c4/containers.md

```diff
@@ -0,0 +1,37 @@
+```mermaid
+C4Context
+  title [Container] FloPy
+
+  Boundary(mf6, "MODFLOW 6"){
+    System(src, "Source code",)
+    System(dfn, "Definition files",)
+  }
+
+  Boundary(flopy, "FloPy") {
+
+    Boundary(devs, "Developer APIs") {
+      Person(dev, "Developer", "")
+      System(fpycodegen, "Code generation")
+
+      Boundary(users, "User APIs") {
+          Person(user, "User", "")
```

Comment:
Consider moving the users and developers outside of the boundaries, as they are not part of the boundary itself.